lib/core/monitor_impl.toit

```diff
@@ -49,7 +49,7 @@ class __Monitor__:
     if deadline:
       timer = self.acquire_timer_
       // Arrange for the notify_ method to be called if the timeout expires.
-      timer.arm this deadline
+      timer.set_target this
```

Comment:
Should this be moved into acquire_timer_ so you can't forget it?

Reply:
Done.